src/api/__specs__/charities-spec.js

```diff
@@ -13,6 +13,7 @@ describe('charities', () => {
   };
 
   beforeEach(() => {
+    callback = sinon.spy();
```

Comment:
Is making this a `let`... is this instead of calling `callback.reset()` in beforeEach? (sorry, am positively certain that i'm missing context :)